Assets/Scripts/Core/Controllers/GameController.cs

```diff
@@ -64,6 +64,14 @@ public class GameController : MonoBehaviour {
 			} else {
 				Debug.Log(SceneManager.GetActiveScene().name + " has no level to load!");
 			}
+			if (SceneManager.GetActiveScene().name == "Final Level")
```

Comment:
Coding style violations, no space between `if` and `(`, no newline between `)` and `{`, and this can be an else if in the previous case.